### PR TITLE
Branch export csv

### LINKED ISSUE #61: Export list of volunteers into CSV

### PR DESCRIPTION
**Pull request by SimKianSeng**
# General remarks
Fixes #61.
- :white_check_mark: Feature implemented
- :white_check_mark: JUnit testing
- :white_check_mark: Code quality checks
- :white_check_mark: UG
- :white_check_mark: DG

Ready to be reviewed and merged.

# Feature information
**Command format:** `export`
**Command success message:** `Data successfully exported to data directory.`
**Command failure message:** `Unable to locate the json file.`
**What happens:** data from JSON file will be exported to CSV and will be located in `[JAR file location]/data` directory.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ExportCommand.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ * The files are located in <location of jar file>\data directory.
+ */
+public class ExportCommand extends Command {
+
+    public static final String COMMAND_WORD = "export";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Export the information of all the people and activities in the application to a CSV file.\n";
+
+    public static final String MESSAGE_SUCCESS = "Data successfully exported.";
```
Comment (Benson15912) [inline src/main/java/seedu/address/logic/commands/ExportCommand.java:29]: Maybe can tell the user where to find the exported data.

**Comment by Benson15912** [comment]
Function looks to be working. Just one issue when starting a fresh Addressbook, export will say that filepath is not found.
Other than that, can be more detailed on the success message, like what kind of data is exported and where it can be found

**Review by Benson15912 — APPROVED** [review]
lgtm

**Comment by YowSiaoKang** [comment]
LGTM, however note that when .csv file generated is used by another application, running export will result in error.

**Comment by YowSiaoKang** [comment]
LGTM, but note that when .csv file generated is used by another application, running export will result in error.

**Review by tayziyi — APPROVED** [review]
lgtm, functionality tested, instructions are easy to follow

**Pull request merged**